releases/values/teleport-ent/teleport-ent-auth.yaml.gotmpl

```diff
@@ -123,7 +123,7 @@ config:
       #
       # IMPORTANT: if you change cluster_name, it will invalidate all generated
       # certificates and keys (may need to wipe out /var/lib/teleport directory)
-      cluster_name: "{{ requiredEnv "TELEPORT_PROXY_DOMAIN_NAME" }}"
+      cluster_name: "{{ coalesce (env "TELEPORT_CLUSTER_NAME") (env "STAGE") (requiredEnv "TELEPORT_PROXY_DOMAIN_NAME") }}"
```

Comment:
should `TELEPORT_PROXY_DOMAIN_NAME` be required now since we use `coalesce`?

Reply:
Yes, it is required elsewhere, and one of those 3 env vars must be set, so I require that one.